### PR TITLE
style: various token changes

### PR DESCRIPTION
**Pull request by QuintonJason**
# Description
- [x] Tokens - added zeroes to 2 digit values
- [x] Tokens - apply change to component tokens
- [x] Font-size - correct 2 digit values
- [x] Colors - updated `charcoal` to `grey` mapping

|  Before  |  After  |
|--------|--------|
|Screenshot 2024-08-12 at 11 41 23 AM|Screenshot 2024-08-12 at 12 25 25 PM|

Fixes #(issue)
[DSS-790](https://kajabi.atlassian.net/browse/DSS-790)

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

[DSS-790]: https://kajabi.atlassian.net/browse/DSS-790?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by pixelflips — APPROVED** [review]

**Review by pixelflips — APPROVED** [review]

File: libs/core/src/components/pds-chip/pds-chip.scss [hunk 1]
```diff
@@ -21,7 +21,7 @@
   --color-background-warning-dot: var(--pine-color-yellow-300);
   --color-background-warning-interactive: var(--pine-color-yellow-200);
 
-  --font-size-sm: var(--pine-font-size-087);
+  --font-size-sm: var(--pine-font-size-85);
```
Comment (ju-Skinner) [inline libs/core/src/components/pds-chip/pds-chip.scss]: is there a reason why it's not `085`? It seems all other two digits follow the three-digit placeholder sequence. The long and short is, `pine-085` will sort differently than `pine-85`, due to alphanumeric sorting. `*-085` will show before `*-100`, whereas `*-85` will show afterwords.

File: libs/core/src/components/pds-link/pds-link.scss [hunk 2]
```diff
@@ -7,7 +7,7 @@
   --color-text-default: var(--pine-color-charcoal-400);
 
   --font-size: var(--pine-font-size-100);
-  --font-size-sm: var(--pine-font-size-087);
+  --font-size-sm: var(--pine-font-size-85);
```
Comment (ju-Skinner) [inline libs/core/src/components/pds-link/pds-link.scss]: Here we are using `85` but below we are using `075`, why the inconsistency?

**Review by ju-Skinner — CHANGES_REQUESTED** [review]
There is an inconsistency between the token values for this change. It appears that all tokens should be changed to a non three-digit placeholder if they are only one or two digits.

**Review by anechol — CHANGES_REQUESTED** [review]
Let's discuss these token name changes post-demo.

File: libs/core/src/components/pds-chip/pds-chip.scss [hunk 3]
```diff
@@ -21,7 +21,7 @@
   --color-background-warning-dot: var(--pine-color-yellow-300);
   --color-background-warning-interactive: var(--pine-color-yellow-200);
 
-  --font-size-sm: var(--pine-font-size-087);
+  --font-size-sm: var(--pine-font-size-085);
```
Comment (ju-Skinner) [inline libs/core/src/components/pds-chip/pds-chip.scss:24]: So are all tokens that include digits using three placeholder

File: libs/core/src/components/pds-accordion/pds-accordion.scss [hunk 4]
```diff
@@ -18,10 +18,10 @@ details {
   --number-animation-transform-timing: 200ms;
 
   --spacing-details-padding-inline: var(--pine-spacing-150);
-  --spacing-details-padding-block: var(--pine-spacing-100);
-  --spacing-summary-padding-block: var(--pine-spacing-50);
+  --spacing-details-padding-block-end: calc(var(--pine-spacing-200) + 2) //18 + 2 = 18px;
```
Comment (pixelflips) [inline libs/core/src/components/pds-accordion/pds-accordion.scss:21]: I'd be curious to understand the calculations here vs prev token usage. Not a blocker

File: libs/core/src/components/pds-link/pds-link.scss [hunk 5]
```diff
@@ -1,17 +1,17 @@
 :host {
 
   --border-outline: 4px solid var(--pine-color-blue-200);
-  --border-radius: var(--pine-border-radius-075);
+  --border-radius: var(--pine-border-radius-75);
```
Comment (pixelflips) [inline libs/core/src/components/pds-link/pds-link.scss]: missing 0?

**Review by pixelflips — APPROVED** [review]
Couple of comments, otherwise LGTM!

**Review by anechol — APPROVED** [review]

**Pull request merged**